clamav-mirror/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM python:3.9.9-alpine3.13
+FROM python:3.10.6-alpine3.15
 
-ENV CLAM_VERSION=0.103.6-r0
+ENV CLAM_VERSION=0.104.3-r0
 ENV SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.12/supercronic-linux-amd64 \
     SUPERCRONIC=supercronic-linux-amd64 \
     SUPERCRONIC_SHA1SUM=048b95b48b708983effb2e5c935a1ef8483d9e3e
 
 WORKDIR /clam
 
-RUN apk add --update-cache bash vim wget shadow curl rsync lighttpd \
+RUN apk add --update-cache bash curl rsync lighttpd \
```

Comment:
should this be `--no-cache` or at least have a `&& rm /var/cache/apk/*` ?

Reply:
Thanks for the suggestion, I have added --no-cache